packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/MultiWorkspaceDropdown/internal/MultiWorkspaceDropdownDefaultComponents.tsx

```diff
@@ -0,0 +1,174 @@
+import { useLingui } from '@lingui/react/macro';
+import { useBuildWorkspaceUrl } from '@/domain-manager/hooks/useBuildWorkspaceUrl';
+import { multiWorkspaceDropdownState } from '@/ui/navigation/navigation-drawer/states/multiWorkspaceDropdownState';
+import { NavigationDrawerItem } from '@/ui/navigation/navigation-drawer/components/NavigationDrawerItem';
+import { getSettingsPath } from '~/utils/navigation/getSettingsPath';
+import { SettingsPath } from '@/types/SettingsPath';
+import { useDropdown } from '@/ui/layout/dropdown/hooks/useDropdown';
+import { MULTI_WORKSPACE_DROPDOWN_ID } from '@/ui/navigation/navigation-drawer/constants/MulitWorkspaceDropdownId';
+import { useAuth } from '@/auth/hooks/useAuth';
+import { AppPath } from '@/types/AppPath';
+import { useSignUpInNewWorkspaceMutation } from '~/generated/graphql';
+import { SnackBarVariant } from '@/ui/feedback/snack-bar-manager/components/SnackBar';
+import { useSnackBar } from '@/ui/feedback/snack-bar-manager/hooks/useSnackBar';
+import { DropdownMenuHeader } from '@/ui/layout/dropdown/components/DropdownMenuHeader/DropdownMenuHeader';
+import { Dropdown } from '@/ui/layout/dropdown/components/Dropdown';
+import { SelectHotkeyScope } from '@/ui/input/types/SelectHotkeyScope';
+import { useColorScheme } from '@/ui/theme/hooks/useColorScheme';
+import styled from '@emotion/styled';
+
+const StyledDescription = styled.div`
+  color: ${({ theme }) => theme.font.color.light};
+  padding-left: ${({ theme }) => theme.spacing(1)};
+`;
+
+export const MultiWorkspaceDropdownDefaultComponents = () => {
+  const currentWorkspace = useRecoilValue(currentWorkspaceState);
+  const { t } = useLingui();
+  const { redirectToWorkspaceDomain } = useRedirectToWorkspaceDomain();
+  const workspaces = useRecoilValue(workspacesState);
+  const { buildWorkspaceUrl } = useBuildWorkspaceUrl();
+  const { closeDropdown } = useDropdown(MULTI_WORKSPACE_DROPDOWN_ID);
+  const { signOut } = useAuth();
+  const { enqueueSnackBar } = useSnackBar();
+  const { colorScheme, colorSchemeList } = useColorScheme();
+
+  const [signUpInNewWorkspaceMutation] = useSignUpInNewWorkspaceMutation();
+
+  const setMultiWorkspaceDropdownState = useSetRecoilState(
+    multiWorkspaceDropdownState,
+  );
+
+  const handleChange = async (workspace: Workspaces[0]) => {
+    redirectToWorkspaceDomain(getWorkspaceUrl(workspace.workspaceUrls));
+  };
+
+  const createWorkspace = () => {
+    signUpInNewWorkspaceMutation({
+      onCompleted: (data) => {
+        return redirectToWorkspaceDomain(
+          getWorkspaceUrl(data.signUpInNewWorkspace.workspace.workspaceUrls),
+          AppPath.Verify,
+          {
+            loginToken: data.signUpInNewWorkspace.loginToken.token,
+          },
+          '_blank',
+        );
+      },
+      onError: (error: Error) => {
+        enqueueSnackBar(error.message, {
+          variant: SnackBarVariant.Error,
+        });
+      },
+    });
+  };
+
```

Comment:
The positioning feels a bit off?
<img width="230" alt="Screenshot 2025-03-12 at 11 01 45" src="https://github.com/user-attachments/assets/9a3199c9-bd2f-428a-bd03-dc68c9dd690a" />
I think I would have placed it below the surface that was clicked. It might be a bit strange to see Apple + logo repeated twice though! CC @Bonapara for input

Reply:
Let's position the menu directly above where the Apple logo is located. (Ensure the logo appears stationary when opening the menu.)